src/Data/String/Extra.purs

```diff
@@ -64,6 +66,19 @@ words string =
   else
     asciiWords string
 
+-- | Calculates the levenshtein distance between two strings.
+-- | ```purescript
+-- | levenshtein "book" "back" -- 2
+-- | ```
+foreign import levenshtein :: String -> String -> Int
+
+-- | Calculates the Sørensen-Dice coefficient between two strings.
+-- | ```purescript
+-- | levenshtein "WHIRLED" "WORLD" -- 0.2000
+-- | ```
+foreign import sorensen_dice_coefficient :: String -> String -> Number
```

Comment:
Can we implement these using snake case?

```
sorensenDiceCoefficient`
```

Also, the example still uses "levenshtein" here.

Reply:
Yes ok we can use `camelCase` and not `snake_case`, it stays in the desert anyway. The example function name is wrong but the result is correct. Can you make direct edits instead of comments so i can approve them?